image-builder/image_builder.sh

```diff
@@ -466,12 +467,20 @@ main() {
 		exit 0
 	fi
 
-	if [ -n "${USE_DOCKER}" ] ; then
-		build_with_docker "${rootfs}" "${image}" "${fs_type}" "${block_size}" \
-						  "${root_free_space}" "${agent_bin}" "${agent_init}"
+	if [ -n "${USE_DOCKER}" ]; then
```

Comment:
so, `USE_DOCKER` takes precedence over `USE_PODMAN`, it should be mentioned in the README